src/Mautic.php

```diff
@@ -32,16 +33,24 @@ class Mautic
      */
     protected $cookie;
 
+    /**
+     * Mautic Configuration
+     *
+     * @var MauticConfig
+     */
+    protected $config;
+
     /**
      * Constructor
      *
      * @param string $baseUrl
      */
-    public function __construct($baseUrl)
+    public function __construct($baseUrl, $config)
```

Comment:
```suggestion
    public function __construct($baseUrl, Config $config = null)
```
This would be a BC break